src/static/css/main.less

```diff
@@ -13,7 +13,7 @@
 @import (less) "cf-layout.less";
 @import (less) "cf-buttons.less";
 @import (less) "cf-forms.less";
-@import (less) "cf-notification.less";
+@import (less) "cf-notifications.less";
```

Comment:
Won't this require updating the dependency in `package.json` to be `cf-notifications`?

Reply:
Uh yup, fixed!